hq/app/logging/Cloudwatch.scala

```diff
@@ -75,8 +75,8 @@ object Cloudwatch extends Logging {
     val request = new PutMetricDataRequest().withNamespace(namespace).withMetricData(datum)
 
     Try(cloudwatchClient.putMetricData(request)) match {
-      case Success(response) => logger.info(s"putMetric: ${datum}")
-      case Failure(e) => logger.error(s"putMetric failure: ${datum}", e)
+      case Success(_) => logger.debug(s"putMetric success: $datum")
```

Comment:
This is the change requested in https://trello.com/c/y2SPZYvz/799-%F0%9F%9A%A8-security-hq-delete-noisy-log-lines - I stumbled upon the other problem whilst figuring out what to update!